### ISSUE TITLE
Define XML-only run file format

### ISSUE THREAD

**Issue by senier** [Componolit/componolit#70]
Current Genode run files mix expect code and XML code to implement variants and configuration options. This has a number of drawbacks:

* Hard to read and understand
* Can be checked only after being expanded to XML
* Procedures instead of declaration
* Automatic checks and transformations impossible on source file
* Comments at XML and expect level
* Escaping issues (e.g. need to escape all quotes for variable substitution of XML fragment)

We want to define and XML equivalent of current run files which mitigates those issues. The next step would be a tool (XSLT?) to transform those files to regular run files and a tool to perform rigorous checks on the input (XML Schema?).

**Comment by senier** [comment]
WONTFIX and superseeded by Genode's depot runtime files.
